core/templates/local_vector.h

```diff
@@ -71,6 +81,14 @@ class LocalVector {
 		}
 	}
 
+	void pop_back() {
+		ERR_FAIL_COND(count == 0);
```

Comment:
Why not use `CRASH_BAD_UNSIGNED_INDEX` here too?

Reply:
I remember godot prefer throw error and continue than crash, follow that rule.

Reply:
Ah, right, the other function crashes. Makes sense to me!